elasticsearch/run.sh

```diff
@@ -1,7 +1,11 @@
 #! /bin/bash
 
-sed --in-place=.bak 's/searchguard.ssl.transport.http.enabled: true/searchguard.ssl.transport.http.enabled: false/' /usr/share/elasticsearch/config/elasticsearch.yml
-sed -i 's/searchguard.allow_all_from_loopback: false/searchguard.allow_all_from_loopback: true/' /usr/share/elasticsearch/config/elasticsearch.yml
+mkdir -p /elasticsearch/$CLUSTER_NAME
+ln -s /etc/elasticsearch/keys/searchguard.key /elasticsearch/$CLUSTER_NAME/searchguard_node_key.key
+
+sed --in-place=.bak 's/enabled: true/enabled: false/' /usr/share/elasticsearch/config/elasticsearch.yml
```

Comment:
@ewolinetz  What's the purpose of this - to disable clustering during prehook? Seems like it could be a bit too broad.